cilium/cilium.io
# feat: add lab card to want to learn more section

**Describe what changes this pull request brings**

Add new card (Lab) to Want to Learn More section to Use Case pages

<details>
<summary><b>Screenshots</b></summary>

<img width="759" alt="image" src="https://github.com/user-attachments/assets/33aad316-9b62-4afc-8e50-cfef929d5c0f">

<br />

<img width="508" alt="image" src="https://github.com/user-attachments/assets/88b4f833-5b79-4da7-80ff-877eef51c91e">

<br />

<img width="136" alt="image" src="https://github.com/user-attachments/assets/dda23a03-68b6-489c-98da-03b86641b0fe">

<br />

<img width="1345" alt="image" src="https://github.com/user-attachments/assets/1990db97-40b7-4bdc-a691-c3159ab18328">

<br />

<img width="508" alt="image" src="https://github.com/user-attachments/assets/827e7d51-704e-445a-851e-bd2ac4aaeccc">

<br />

<img width="382" alt="image" src="https://github.com/user-attachments/assets/525f5935-e952-4d7a-ad5d-42e2315a1447">

<br />

<img width="190" alt="image" src="https://github.com/user-attachments/assets/1d1d076d-3f6a-4b68-a505-0392a6a9119e">

</details>

**Steps to check:**

1. Open [preview](https://deploy-preview-555--cilium.netlify.app/use-cases/cni/)
2. Make sure that everything looks and works as expected
